src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/EditorInProcess.cs

```diff
@@ -132,6 +142,242 @@ public async Task SetUseSuggestionModeAsync(bool value, CancellationToken cancel
             }
         }
 
+        #region Navigation bars
+
+        public async Task ExpandProjectNavigationBarAsync(CancellationToken cancellationToken)
+        {
+            await ExpandNavigationBarAsync(index: 0, cancellationToken);
+        }
+
+        public async Task ExpandTypeNavigationBarAsync(CancellationToken cancellationToken)
+        {
+            await ExpandNavigationBarAsync(index: 1, cancellationToken);
+        }
+
+        public async Task ExpandMemberNavigationBarAsync(CancellationToken cancellationToken)
+        {
+            await ExpandNavigationBarAsync(index: 2, cancellationToken);
+        }
+
+        public async Task ExpandNavigationBarAsync(int index, CancellationToken cancellationToken)
+        {
+            await TestServices.Workspace.WaitForAsyncOperationsAsync(FeatureAttribute.NavigationBar, cancellationToken);
+
+            await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
+
+            var view = await GetActiveTextViewAsync(cancellationToken);
+            var combobox = (await GetNavigationBarComboBoxesAsync(view, cancellationToken))[index];
+            FocusManager.SetFocusedElement(FocusManager.GetFocusScope(combobox), combobox);
+            combobox.IsDropDownOpen = true;
+        }
+
+        public async Task<ImmutableArray<string>> GetProjectNavigationBarItemsAsync(CancellationToken cancellationToken)
+        {
+            return await GetNavigationBarItemsAsync(index: 0, cancellationToken);
+        }
+
+        public async Task<ImmutableArray<string>> GetTypeNavigationBarItemsAsync(CancellationToken cancellationToken)
+        {
+            return await GetNavigationBarItemsAsync(index: 1, cancellationToken);
+        }
+
+        public async Task<ImmutableArray<string>> GetMemberNavigationBarItemsAsync(CancellationToken cancellationToken)
+        {
+            return await GetNavigationBarItemsAsync(index: 2, cancellationToken);
+        }
+
+        public async Task<ImmutableArray<string>> GetNavigationBarItemsAsync(int index, CancellationToken cancellationToken)
```

Comment:
make private?

Reply:
➡️ Eliminated the intermediate callers and left this public